PetApi/Controllers/PetController.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection.Metadata.Ecma335;
+using System.Xml.Linq;
+using Microsoft.AspNetCore.Mvc;
+using PetApiTest.Controller;
+
+namespace PetApi.Controllers
+{
+    [ApiController]
+    [Route("api")]
+    public class PetController
+    {
+        private static List<Pet> pets = new List<Pet>();
+
+        [HttpPost("addNewPet")]
+        public Pet AddNewPet(Pet addPet)
+        {
+            pets.Add(addPet);
+            return addPet;
+        }
+
+        [HttpPost("addNewPets")]
+        public List<Pet> AddNewPets(List<Pet> addPets)
+        {
+            pets.AddRange(addPets);
+            return pets;
+        }
+
+        [HttpGet("getAllPets")]
+        public List<Pet> GetAllPets()
+        {
+            return pets;
+        }
+
+        [HttpDelete("deleteAllPets")]
+        public List<Pet> DeleteAllPets()
+        {
+            pets.Clear();
+            return pets;
+        }
+
+        [HttpGet("findPetByName")]
+        public Pet FindPetByName([FromQuery] string name)
+        {
+            return pets.Find(pet => pet.Name == name);
+        }
+
+        [HttpGet("findPetsByType")]
+        public List<Pet> FindPetsByType([FromQuery] string type)
+        {
+            var findResult = pets.FindAll(pet => pet.Type.Equals(type));
+            return findResult;
+        }
+
+        [HttpGet("findPetsByPriceRange")]
+        public List<Pet> FindPetsByPriceRange([FromQuery] string min, [FromQuery] string max)
```

Comment:
why define a string min price and max price here?   why don't use a int directly